package.json

```diff
@@ -24,7 +30,7 @@
     "npm:login": "npm login --registry https://registry.npmjs.org/",
     "publish": "lerna publish from-package --ignore-scripts --yes --registry https://registry.npmjs.org/",
     "debug": "karma start karma.config.js --browsers Chrome",
-    "karma": "karma start karma.config.js --single-run --browsers ChromeHeadless,FirefoxHeadless"
```

Comment:
Why remove Firefox?

Reply:
It wasn't explicitly being listed in karma.config.js, where `ChromeHeadless` was. I can re-add it